exonum/src/blockchain/config.rs

```diff
@@ -62,6 +62,8 @@ pub struct ConsensusConfig {
     pub peers_timeout: Milliseconds,
     /// Maximum number of transactions per block.
     pub txs_block_limit: u32,
+    /// Maximum message length (in bytes).
+    pub max_message_len: usize,
```

Comment:
Let's make it `u32`? `ConsensusConfig` is serialize, so it would be nice if it didn't depend on bitness.

Reply:
Sounds reasonable, np